spec/requests/api/v1/topics/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe 'GET api/v1/topics', type: :request do
+  let(:password)  { 'password' }
+  let(:user)      { create(:user, password: password) }
+  let!(:topic)    { create(:topic) }
+
+  it 'returns the topic' do
+    get '/api/v1/topics', headers: auth_headers, as: :json
```

Comment:
Use the path helpers for this path provided by rails. Check https://guides.rubyonrails.org/routing.html#path-and-url-helpers